include/predicatesDLLExport.h

```diff
@@ -0,0 +1,45 @@
+
+
+#ifndef _predicatesdllexport_h_
+#define _predicatesdllexport_h_
+
+
+#if defined (_MSC_VER)
+//#pragma warning(disable: 4267)
+//#pragma warning(disable: 4800) /*  warning C4800: 'double' : forcing value to bool 'true' or 'false' */
+#endif
+
+
+/* Cmake will define predicates_EXPORTS on Windows when it
+configures to build a shared library. If you are going to use
+another build system on windows or create the visual studio
+projects by hand you need to define predicates_EXPORTS when
+building the predicates DLL on windows.
+*/
+
+
+#if defined (predicates_EXPORTS)  /* Compiling the predicates DLL/Dylib */
+  #if defined (_MSC_VER)  /* MSVC Compiler Case */
+    #define  predicates_EXPORT __declspec(dllexport)
```

Comment:
Can you change this to `PREDICATES_EXPORT`? I usually stick to all-caps for preprocessor defines.